jupyterhub/jupyterhub/overlays/build/jupyterhub-img-imagestream.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: image.openshift.io/v1
-kind: ImageStream
+kind: DockerImage
```

Comment:
This looks like a typo since `DockerImage` doesn't exist on `image.openshift.io`

Reply:
Yeah sorry, that was a typo.